cmd/axon/schema.go

```diff
@@ -73,3 +76,74 @@ func getPrimaryKeyForChange(change *warppipe.Changeset) ([]string, error) {
 	}
 	return col, nil
 }
+
+func loadSerialColumns(conn *sqlx.DB) error {
+	var rows []struct {
+		TableName     string `db:"table_name"`
+		ColumnName    string `db:"column_name"`
+		ColumnDefault string `db:"column_default"`
+	}
+	err := conn.Select(&rows, `
+	SELECT
+		table_name,
+		column_name,
+		column_default
+	FROM information_schema.columns
+	WHERE
+		table_schema = 'public'
+	AND
+		column_default LIKE 'nextval(%'`,
+	)
+	if err != nil {
+		return fmt.Errorf("loadSerialColumns: %w", err)
+	}
+
+	for _, r := range rows {
+		colDefault := strings.Split(r.ColumnDefault, "'")
+		sequenceName := colDefault[1]
```

Comment:
Im guessing it is safe not to check the length?

Reply:
There's a lot of additional error checking needed in this entire codebase, but, yes, unless there is a database is broken this should never fail.